test/one/profiler/test/TestProcess.java

```diff
@@ -261,7 +261,7 @@ public Output profile(String args, boolean sudo) throws IOException, TimeoutExce
                 .redirectError(createTempFile(PROFERR))
                 .start();
 
-        waitForExit(p, 10);
+        waitForExit(p, 20);
```

Comment:
Why did the waiting time increase?

Reply:
for the change below to help the test to pass on Mac